src/Groups/pcgroup.jl

```diff
@@ -0,0 +1,345 @@
+export collector,
+       pc_group,
+       set_conjugate!,
+       set_power!,
+       set_relative_order!,
+       set_relative_orders!
+
+############################################################################
+
+# Create GAP filters that describe
+# - the union of `IsMultiplicativeElementWithInverseByPolycyclicCollector`
+#   and `IsPcpElement` and
+# - the union of `IsPcGroup` and `IsPcpGroup`.
+# This must be done at runtime.
+function __init_PcGroups()
+  GAP.evalstr("DeclareFilter(\"IsPcElementOrPcpElement\")");
+  GAP.evalstr("InstallTrueMethod(IsPcElementOrPcpElement, IsMultiplicativeElementWithInverseByPolycyclicCollector)");
+  GAP.evalstr("InstallTrueMethod(IsPcElementOrPcpElement, IsPcpElement)");
+  GAP.evalstr("BindGlobal(\"IsPcGroupOrPcpGroup\", IsGroup and CategoryCollections(IsPcElementOrPcpElement))");
```

Comment:
Could also do tis (but do whatever you prefer, though)
```suggestion
  GAP.evalstr("""
      DeclareFilter("IsPcElementOrPcpElement");
      InstallTrueMethod(IsPcElementOrPcpElement, IsMultiplicativeElementWithInverseByPolycyclicCollector);
      InstallTrueMethod(IsPcElementOrPcpElement, IsPcpElement);
      BindGlobal("IsPcGroupOrPcpGroup", IsGroup and CategoryCollections(IsPcElementOrPcpElement));
      """)
```